deploy-service/common/src/main/java/com/pinterest/deployservice/bean/BuildBean.java

```diff
@@ -36,6 +36,7 @@
  * publish_date     BIGINT              NOT NULL,
  * publisher        VARCHAR(64),
  * scm_info         VARCHAR(512),
+ * ensure_trusted_build  TINYINT(1)    DEFAULT 0
```

Comment:
for build table, i think we don't need this field.  the enforcement is from environment table. so when an environment is set to ensure_trusted_build, the build url need to be from the trusted_build_url

Reply:
Thanks for pointing out. updated.